content/en/monitors/incident_management.md

```diff
@@ -0,0 +1,84 @@
+
+In the Datadog paradigm, any of the following are appropriate situations for declaring an incident:
+
+* Something is happening that will affect customers.
+* Something is happening that you need help with.
+* You are unsure whether you should call an incident.
+
+## Usage
+
+Incident Management requires no installation. To view your incidents, go to the [Incidents][1] page for a feed of all ongoing incidents.
+
+### Creating an incident
+
+You can declare an incident directly from a graph. You can also go to the upper right hand side of the Incidents page and click on “New Incident”. The following dialog box appears.
+
+{{< img src="monitors/incidents/create-1.png" alt="New Incident"  style="width:80%;">}}
+
+**Severity**: Denotes the severity of your incident, from SEV-1 (most severe) to SEV-5 (least severe). If the severity of your incident is unknown, select UNKNOWN.
+
+* SEV-1: Critical impact
+* SEV-2: High impact
+* SEV-3: Moderate impact
+* SEV-4: Low impact
+* SEV-5: Informational
+* UNKNOWN: Unknown
+
+**Title**: Give your incident a descriptive title.
+
+**Signals**: The reason(s) you are declaring the incident. These can be graphs, logs, or other key visuals.
+
+**Incident commander**: This person is assigned as the leader of the incident investigation.
+
+**Additional notifications**: Notify other teams or people.
+
+Click on “Declare Incident” to finish creating your incident.
+
+### Example workflow
+
+#### 1. Discover an issue
+
+Consider a scenario in which you are looking at a dashboard, and you notice that one particular service is showing an especially high error count. Using the Export button in the upper right of a widget, you can declare an incident.
+
+{{< img src="monitors/incidents/workflow-1-graph.png" alt="From Graph"  style="width:80%;">}}
+
+#### 2. Declare an incident and assemble your team
+
+Use the New Incident modal to assemble your team and notify them. The graph from which you created the incident is automatically attached as a signal. Attach any other signals that would give your team the necessary context to begin resolving this issue. The Slack and PagerDuty integrations enable you to send notifications through those services.
+
+{{< img src="monitors/incidents/workflow-2-modal.png" alt="New Incident"  style="width:60%;">}}
+
+#### 3. Communicate and begin troubleshooting
+
+If you have the [Datadog Slack app][2] installed, the Slack integration can automatically create a new channel dedicated to the incident, so you can consolidate communication with your team and begin troubleshooting.
+
+{{< img src="monitors/incidents/workflow-3-slack.png" alt="Communicate"  style="width:80%;">}}
+
+#### 4. Update incident status
+
+After resolving the issue, update the incident. Set the status to "Stable" to indicate the problem has been mitigated, and set the customer impact field so that your organization knows how this issue has affected customers.
```

Comment:
Update the incident as the situation evolves. Set the status to "Stable" to indicate the problem has been mitigated, and set the customer impact field so that your organization knows how this issue has affected customers. Set the status to "Resolved" once the incident is completely fixed.

Reply:
^ For line 71.